src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/AnnotationLineMapper.java

```diff
@@ -196,137 +196,27 @@ private void parseExtraField(ConsequenceType consequenceType, String extraField,
-                                "phase_1_ASN", currentAnnotation));
-
-                    break;
                 case "biotype":
                     consequenceType.setBiotype(keyValue[1]);
                     break;
-//                case "canonical":
-//                    variantEffect.setCanonical(keyValue[1].equalsIgnoreCase("YES") || keyValue[1].equalsIgnoreCase("Y"));
-//                    break;
-//                case "ccds":
-//                    variantEffect.setCcdsId(keyValue[1]);
-//                    break;
-//                case "cell_type":
-//                    variantAnnotation.getRegulatoryEffect().setCellType(keyValue[1]);
-//                    break;
-//                case "clin_sig":
-//                    variantEffect.setClinicalSignificance(keyValue[1]);
-//                    break;
-//                case "distance":
-//                    variantEffect.setVariantToTranscriptDistance(Integer.parseInt(keyValue[1]));
-//                    break;
-//                case "domains":
-//                    variantEffect.setProteinDomains(keyValue[1].split(","));
-//                    break;
-                case "ea_maf":
-                        if(currentAnnotation.getPopulationFrequencies()==null) {
-                            currentAnnotation.setPopulationFrequencies(new ArrayList<>());
-                        }
-                        currentAnnotation.getPopulationFrequencies().add(parsePopulationFrequency(keyValue[1], "ESP_6500",
-                                "European_American", currentAnnotation));
-
-                    break;
-//                case "ensp":
-//                    variantEffect.setProteinId(keyValue[1]);
-//                    break;
-                case "eur_maf":
-                        if(currentAnnotation.getPopulationFrequencies()==null) {
-                            currentAnnotation.setPopulationFrequencies(new ArrayList<>());
-                        }
-                        currentAnnotation.getPopulationFrequencies().add(parsePopulationFrequency(keyValue[1], "1000GENOMES",
-                                "phase_1_EUR", currentAnnotation));
-
-                    break;
-//                case "exon":
-//                    variantEffect.setExonNumber(keyValue[1]);
-//                    break;
-                case "gmaf": // Format is GMAF=G:0.2640  or  GMAF=T:0.1221,-:0.0905
-                        if(currentAnnotation.getPopulationFrequencies()==null) {
-                            currentAnnotation.setPopulationFrequencies(new ArrayList<>());
-                        }
-                        currentAnnotation.getPopulationFrequencies().add(parsePopulationFrequency(keyValue[1], "1000GENOMES",
-                                "phase_1_ALL", currentAnnotation));
-
-                    break;
                 case "hgvsc":
-                    if(currentAnnotation.getHgvs()==null) {
```

Comment:
If this is no longer going to be checked then it must be properly initialized in the object constructor. Same with the rest of collection-based attributes, which currently are all null right after construction.